packages/flutter_test/lib/src/binding.dart

```diff
@@ -1271,12 +1305,13 @@ enum LiveTestWidgetsFlutterBindingFramePolicy {
   /// This is used for running the test on a device, where scheduling of new
   /// frames respects what the engine and the device needed.
   ///
-  /// Compared to `fullyLive` this policy ignores the frame requests from pump
+  /// Compared to [fullyLive] this policy ignores the frame requests from [pump]
   /// of the test code so that the frame scheduling respects the situation of
-  /// that for the real environment, and avoids waiting for the new frame beyond
-  /// the expected time.
+  /// that for the real environment, and avoids waiting for the artificially
```

Comment:
I'm having trouble unpacking this sentence:
```
Compared to [fullyLive] this policy ignores the frame requests from [pump]
of the test code so that the frame scheduling respects the situation of
that for the real environment, and avoids waiting for the artificially
[pump]ed frame frame when driving the test in methods like
[WidgetTester.handlePointerEventRecord] or [WidgetTester.fling].
```

??? Perhaps the following?

```
Compared to [fullyLive], this policy ignores frame requests from [pump] 
so that frame scheduling mimics that of the real environment,
and avoids waiting for an artificially[pump]ed frame.
(For example, when driving the test in methods like
[WidgetTester.handlePointerEventRecord] or [WidgetTester.fling].)
```

Reply:
Modified to your version.